### PR TITLE
[Maintenance] Clean up css custom properties

### PR DESCRIPTION
**Pull request by kyledurand**
### WHY are these changes introduced?

Part of https://github.com/Shopify/polaris-react/issues/4732

This also gets us closer to visual parity with the main branch

## 

### How to 🎩

🖥 [Local development instructions](https://github.com/Shopify/polaris-react/blob/master/README.md#development)
🗒 [General tophatting guidelines](https://github.com/Shopify/polaris-react/blob/master/documentation/Tophatting.md)
📄 [Changelog guidelines](https://github.com/Shopify/polaris-react/blob/master/documentation/Versioning%20and%20changelog.md)

<details>
<summary>Copy-paste this code in <code>playground/Playground.tsx</code>:</summary>

```jsx
import React from 'react';
import {Page} from '../src';

export function Playground() {
  return (
    <Page title="Playground">
      {/* Add the code you want to test in here */}
    </Page>
  );
}
```

</details>

### 🎩 checklist

- [x] Tested on [mobile](https://github.com/Shopify/polaris-react/blob/master/documentation/Tophatting.md#cross-browser-testing)
- [ ] Tested on [multiple browsers](https://help.shopify.com/en/manual/intro-to-shopify/shopify-admin/supported-browsers)
- [ ] Tested for [accessibility](https://github.com/Shopify/polaris-react/blob/master/documentation/Accessibility%20testing.md)
- [ ] Updated the component's `README.md` with documentation changes
- [ ] [Tophatted documentation](https://github.com/Shopify/polaris-react/blob/master/documentation/Tophatting%20documentation.md) changes in the style guide
- [ ] For visual design changes, ping @ sarahill to update the Polaris UI kit

### CODE REVIEW

File: src/components/Avatar/Avatar.scss [hunk 1]
```diff
@@ -1,9 +1,9 @@
 @import '../../styles/common';
 
 .Avatar {
-  --pc-avatar-extra-small-size: var(--p-space-6);
-  --pc-avatar-small-size: var(--p-space-8);
-  --pc-avatar-medium-size: var(--p-space-10);
+  --pc-avatar-extra-small-size: 24px;
```
Comment (aveline) [inline src/components/Avatar/Avatar.scss:4]: I'm not sure these sizes should be coupled with the spacing scale. If we made adjustments to the spacing scale, would we also expect avatar size to change?

**Review by aveline — APPROVED** [review]
Some of the nav visual changes were intentional from replacing the hardcoded values with tokens, but I like your solution for still using tokens but getting it closer to main! 💡 👍

File: src/components/Avatar/Avatar.scss (hunk 1 above)
Comment (kyledurand) [reply src/components/Avatar/Avatar.scss:4]: I don't think so, that's why I decoupled them. Might make our coverage worse but I think they should just be hard coded 👍

File: src/tokens/_shape.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  'border-radius-1': '4px',
+  'border-radius-2': '8px',
+  'border-radius-3': '12px',
+  'border-radius-4': '16px',
+  'border-radius-5': '20px',
+  'border-radius-6': '30px',
+  'border-radius-base': '3px',
+  'border-radius-large': '6px',
+  'border-radius-half': '50%',
+};
```
Comment (kyledurand) [inline src/tokens/_shape.ts]: Looks like this file snuck in by accident. I'll remove

File: src/components/Avatar/Avatar.scss (hunk 1 above)
Comment (aveline) [reply src/components/Avatar/Avatar.scss:4]: Ha my bad I was reading the diff backwards! But yes agree! 👍

File: src/components/RangeSlider/components/DualThumb/DualThumb.scss [hunk 3]
```diff
@@ -134,15 +133,15 @@ $range-output-spacing: 16px;
   transition-property: opacity, visibility, bottom;
   transition-duration: var(--p-duration-150);
   transition-timing-function: var(--p-ease);
-  transform: translateX(calc(-50% + #{var(--pc-range-slider-thumb-size) / 2}));
+  transform: translateX(calc(-50% + var(--pc-range-slider-thumb-size) / 2));
```
Comment (kyledurand) [inline src/components/RangeSlider/components/DualThumb/DualThumb.scss]:
```suggestion
  transform: translateX(calc(-50% + (var(--pc-range-slider-thumb-size) / 2)));
```

**Pull request merged**